test/truncate.jl

```diff
@@ -38,13 +38,14 @@ function verify_and_test_drive(jsonfile, selected, n_tsamples::Int,lower::Int,up
 
         println("    testing Truncated($(ex),$lower,$upper)")
         d = Truncated(eval(Meta.parse(ex)),lower,upper)
-        if dtype != TruncatedNormal
-            @assert isa(dtype, Type) && dtype <: UnivariateDistribution
-            @test isa(d, dtypet)
+        if dtype != Uniform # Uniform is truncated to Uniform
```

Comment:
just a minor comment, is this part of the tests required? The one in truncated_uniform.jl seems sufficient

Reply:
I was trying to be conservative and only disable what was causing a problem.

Reply:
is it still required now if the bug is gone?

Reply:
Sorry, I don't know which bug you are referring to.

But if I allow these tests for Uniform then on this branch the test suite fails. Did something change in the meantime?

Reply:
sorry, I thought the modification of the test was to debug something. All good